packages/core-flows/src/definition/cart/steps/get-variant-price-sets.ts

```diff
@@ -0,0 +1,65 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IPricingModuleService, RegionDTO } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+interface StepInput {
+  variantIds: string[]
+  // TODO: Expand the context passed to calculatePrices
+  region: RegionDTO
+}
+
+export const getVariantPriceSetsStepId = "get-variant-price-sets"
+export const getVariantPriceSetsStep = createStep(
```

Comment:
@riqwan, would appreciate your eyes on this implementation. Is there a more appropriate way of retrieving variant prices?

Please ignore the context for now. This will be revisited later.